client/my-sites/site-settings/form-jetpack-scan.jsx

```diff
@@ -282,10 +282,10 @@ module.exports = React.createClass( {
 					disabled={ this.disableForm() } />
 
 				<FormSettingExplanation>
-					{ this.translate( 'Password is not required if you use public key authentication.' ) }
-					<a onClick={ this._onShowDialog } href="#">
-						{ this.translate( 'Get your public key string.' ) }
-					</a>
+					{ this.translate(
```

Comment:
I'll admit I haven't run the code, but looking at it, it doesn't seem valid, you close the parenthesis for `translate` too soon (added whitespace to illustrate it better):
```
{
	this.translate(
		'Password is not required if you use public key authentication. {{a}}Get your public key string.{{/a}}'
	),
	{
		components: { a: <a onClick={ this._onShowDialog } href="#" /> }
	}
}
```
It should something like:
```
{
	this.translate(
		'Password is not required if you use public key authentication. {{a}}Get your public key string.{{/a}}',
		{
			components: { a: <a onClick={ this._onShowDialog } href="#" /> }
		}
	)
}
```